### DISCUSSION TITLE
[CX] 메시지 창 위치 이동이 가능하면 좋겠어요.

### DISCUSSION THREAD

**Discussion by joorijung** [metamong-friends/metamong-friends.github.io#5] [제안하기]
메시지 창을 오른쪽으로 이동시켜 놓고 싶은데, 가운데에만 떠있어서 아바타를 가려서 동시에 보기가 조금 어려운 것 같아요.

**Comment by shlee303** [comment]
자유 이동 혹은 좌/우/중앙 이런식으로 옵션을 줄 수 있을지 검토해보겠습니다 :) 좋은 의견 감사합니다!!!
